composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "reload/github-security-jira",
+    "description": "Create Jira tickets for GitHub security alerts",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.2.0",
+        "symfony/console": "^4",
+        "lesstif/php-jira-rest-client": "^1",
+        "symfony/yaml": "^5.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "GitHubSecurityJira\\": "src/"
+        }
+    },
+    "bin": [
+        "ghsec-jira"
+    ],
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.0"
```

Comment:
Maybe you'd be interested in https://github.com/appocular/coding-standard ? Much more opinionated than PSR12, but I like it (I ought to, don't I?).

Reply:
I went with straight PSR-12 for now :)